tests/app/questionnaire/routing/test_when_rule_evaluator.py

```diff
@@ -111,6 +111,28 @@ def test_answer_source(answer_value, expected_result):
     )
 
 
+@pytest.mark.parametrize(
+    "answer_value, expected_result",
+    [(["India", "Malta"], True), (["india", "malta"], True)],
+)
+def test_answer_source_tuple(answer_value, expected_result):
+    when_rule_evaluator = get_when_rule_evaluator(
+        answer_store=AnswerStore([{"answer_id": "some-answer", "value": answer_value}]),
+    )
+
+    assert (
+        when_rule_evaluator.evaluate(
+            rule={
+                Operator.ANY_IN: [
+                    {"source": "answers", "identifier": "some-answer"},
+                    ("India", "Malta"),
+                ]
+            }
+        )
+        is expected_result
+    )
```

Comment:
The test should be in `test_operators.py` without the usage of when rule evaluator, as it is not dependent on the when rule evaluator.

Reply:
Moved to `test_operators.py`, test is much smaller now. Thanks!